paws.common/R/cache.R

```diff
@@ -3,17 +3,19 @@ ini_cache <- new.env(parent = emptyenv())
 os_env_cache <- new.env(parent = emptyenv())
 
 set_os_env_cache <- function() {
-  env <- system("printenv", intern = TRUE)
+  env <- Sys.getenv()
+
+  # only cache AWS_* environment variables to avoid caching sensitive information
```

Comment:
lmk if you disagree with this! personally, I felt a bit uneasy that _all_ env vars would be cached upon load without input from this user, so this felt better scoped to only the env vars relevant to this package:

<img width="761" alt="image" src="https://github.com/paws-r/paws/assets/56164984/cd57e0d1-3cca-4271-b63f-5c64d0750a00">